playbooks/roles/nginx-proxy/templates/nginx-proxy.j2

```diff
@@ -35,5 +53,9 @@ server {
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "upgrade";
         {% endif %}
+
```

Comment:
Should I leave a code comment explaining why we are handling these cases differently?